ui/components/app/confirm-page-container/confirm-page-container-header/confirm-page-container-header.component.js

```diff
@@ -64,7 +72,19 @@ export default function ConfirmPageContainerHeader({
             </span>
           </div>
         )}
-        {isFullScreen ? null : <NetworkDisplay />}
+        {isFullScreen ? null : (
+          <PickerNetwork
```

Comment:
We can update the underlying html element from a button to a static div using the polymorphic `as` prop
```suggestion
          <PickerNetwork
            as="div"
```

<img width="643" alt="Screenshot 2023-09-11 at 7 42 26 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/d3728ec0-ad2f-4b8e-9abc-02b7ed6da62c">